libs/sdk-ui-theme-provider/src/ThemeProvider/tests/ThemeProvider.test.tsx

```diff
@@ -0,0 +1,139 @@
+// (C) 2020 GoodData Corporation
+import React from "react";
+import { act } from "@testing-library/react";
```

Comment:
We could probably import this directly from `react-dom/test-utils` and drop the extra `@testing-library/react` dependency.